deploy/routeros.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env sh
+
+#Here is a script to deploy cert to routeros router.
+
+#returns 0 means success, otherwise error.
+
+########  Public functions #####################
+
+#domain keyfile certfile cafile fullchain
+routeros_deploy() {
+  _cdomain="$1"
+  _ckey="$2"
+  _ccert="$3"
+  _cca="$4"
+  _cfullchain="$5"
+
+  _debug _cdomain "$_cdomain"
+  _debug _ckey "$_ckey"
+  _debug _ccert "$_ccert"
+  _debug _cca "$_cca"
+  _debug _cfullchain "$_cfullchain"
+
+  if [ -z "$ROUTER_OS_HOST" ]; then
+    _debug "Using _cdomain as ROUTER_OS_HOST, please set if not correct."
+    ROUTER_OS_HOST="$_cdomain"
+  fi
+
+  if [ -z "$ROUTER_OS_USERNAME" ]; then
+    _err "Need to set the env variable ROUTER_OS_USERNAME"
+    return 1
+  fi
+
+  _info "Trying to push key '$_ckey' to router"
+  scp "$_ckey" "$ROUTER_OS_USERNAME@$ROUTER_OS_HOST:$_cdomain.key"
```

Comment:
why use `scp` ?
why not just run acme.sh on the router, and just use `cp` there ?

Reply:
I didn't write the script but I imagine this is for the same reason you don't run `acme.sh` on every machine you want to have an SSL cert.  The same reason you have an SSH deploy script.  I wouldn't run acme on my router because I don't want my router doing things that it doesn't _have_ to do

Reply:
> why use `scp` ?
> why not just run acme.sh on the router, and just use `cp` there ?

Because Mikrotik routers don’t have `bash`, or any other “real” shell.

Reply:
@Neilpang, it's like @kg4zow said, Mikrotik does not have a proper shell, and a lot of their routers have no more than 16MB flash and a bit more ram. (https://mikrotik.com/product/RB960PGS). They do have SSH interface, but no regular commands.

Reply:
if it's just a deploy via ssh.  I think it can be accomplished by the ssh_deploy hook.
Is that true ?

Reply:
No, it can't be accomplished with the ssh_deploy hook, if it could, this PR (thats over a year old) wouldn't continue getting attention. 

@palhaland there are new conflicts that keep popping up while @Neilpang delays merging